Packs/PaloAltoNetworks_IoT3rdParty/Scripts/SendAllPANWIoTAssetsToSIEM/SendAllPANWIoTAssetsToSIEM.yml

```
@@ -0,0 +1,45 @@
+args:
+- auto: PREDEFINED
+  description: Type of asset.
+  name: asset_type
+  predefined:
+  - device
+  - alert
+  - vulnerability
+  required: true
+- defaultValue: PANW IoT 3rd Party SIEM Integration Instance
+  description: Name of the configured syslog sender instance
+  name: syslog_sender_instance
+  required: true
+- default: true
+  defaultValue: PANW IoT 3rd Party Integration Instance
+  description: Name of the configured PANW Iot 3rd Party instance.
+  name: panw_iot_3rd_party_instance
+  required: true
+comment: This automation retrieves all specified assets from the PANW IoT cloud and
+  send them to the SIEM server.
+commonfields:
+  id: SendAllPANWIoTAssetsToSIEM
+  version: -1
+dependson:
+  must:
+  - Palo Alto Networks IoT 3rd Party|||panw-iot-3rd-party-get-asset-list
+  - Palo Alto Networks IoT 3rd Party|||panw-iot-3rd-party-report-status-to-panw
+  - Palo Alto Networks IoT 3rd Party|||panw-iot-3rd-party-convert-assets-to-external-format
```

why coerce the brand? usually the command name is sufficient.
```suggestion
  - |||panw-iot-3rd-party-get-asset-list
  - |||panw-iot-3rd-party-report-status-to-panw
  - |||panw-iot-3rd-party-convert-assets-to-external-format
```